apps/user-portal/src/index.jsp

```diff
@@ -30,34 +30,52 @@
         <title><%= htmlWebpackPlugin.options.title %></title>
 
         <script>
-            function getTenantDomain() {
-                var url = window.location.href;
-                if (url.includes(<%= htmlWebpackPlugin.options.tenantDelimiter %>)){
-                    var index = url.indexOf(<%= htmlWebpackPlugin.options.tenantDelimiter %>);
-                    var endIndex = url.indexOf("/", index+3);
-                    var domain = (endIndex != -1) ? url.substring(index+3, endIndex) : url.substring(index+3);
-                    return domain;
-                }
-                return null;
-            }
+            var getTenantPrefix = function(tenantName) {
+                return  "<%= htmlWebpackPlugin.options.tenantPrefix %>";
+            };
```

Comment:
We can remove this function. And assign the value directly as we are not re-using anywhere else.